docs/user-guide/getting-started.md

```diff
@@ -11,175 +11,189 @@ The Ambassador Edge Stack is designed to run in Kubernetes for production. The m
 
 * Kubernetes 1.11 or later
 * The `kubectl` command-line tool
+* [Edge Control](/reference/edgectl-download)
 
-## Install the Ambassador Edge Stack
+## Quick Install (Recommended!)
 
-The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally.
+The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally. Or, if you're a Minikube user, [check out these directions](#minikube-users).
 
-1. In your terminal, run the following command:
+When you use Edge Control on your publicly
+accessible cluster, it will:
 
-    ```bash
-    kubectl apply -f https://www.getambassador.io/yaml/aes-crds.yaml && \
-    kubectl wait --for condition=established --timeout=90s crd -lproduct=aes && \
-    kubectl apply -f https://www.getambassador.io/yaml/aes.yaml && \
-    kubectl -n ambassador wait --for condition=available --timeout=90s deploy -lproduct=aes
-    ```
+1. Install the Ambassador Edge Stack
+2. Generate a domain name for you to access the Edge Policy Console and complete
+   advanced configuration
+3. Obtain a TLS certificate for that domain name
+4. Configure automatic TLS and HTTPS using that certificate
 
-2. Determine the IP address of your cluster by running the following command:
+**To install the Ambassador Edge Stack:**
 
-    ```bash
-    kubectl get -n ambassador service ambassador -o 'go-template={{range .status.loadBalancer.ingress}}{{print .ip "\n"}}{{end}}'
-    ```
+1. Download the `edgectl`file  for your operating system following [these instructions](/reference/edgectl-download).
+2. Move the file into your PATH (for Windows users, move it into the Windows
+   Systems parth).
+   * If you need to, print your PATH with `echo $PATH`
+3. Ensure the file is executable with the command `chmod a+x /usr/local/bin/edgectl`
+4. Run the executable file with the command `./edgectl`
+5. Now, run the following command: `edgectl install`
 
-    Your load balancer may take several minutes to provision your IP address. Repeat the provided command until you get an IP address.
+Your terminal will print something similar to the following:
 
-    Note: If you are a **Minikube user**, Minikube does not natively support load balancers. Instead, use `minikube service list`. You should see something similar to the following:
+ ```shell
+ $ edgectl install
+ -> Installing the Ambassador Edge Stack 1.0.
```

Comment:
We should use %version% (or whatever the shorthand is) instead of hard-coding 1.0.